test/integration/default/default_test.rb

```diff
@@ -0,0 +1,16 @@
+# InSpec test for recipe habitat::default
```

Comment:
You can probably delete this whole file

Reply:
I"m not even sure how that got in there but I will take care of it now

Reply:
Removed